### PR TITLE
Des 510

### PR DESCRIPTION
**Pull request by JasonLantz**
this installs jest as the test runner and code coverage...

run `npm test`

### CODE REVIEW

**Comment by StommePoes** [comment]
should the copy-utils line be
```
"copy-utils": "cp ./node_modules/@pearson-components/npm-scripts/npm_scripts/*.js ./npm_scripts"
```
or is what I have outdated?

**Comment by wyseguyonline** [comment]
@StommePoes 

it should be:

"copy-utils": "node ./node_modules/@pearson-components/npm-scripts/npm_scripts/cross-platform.js"

File: demo/translations/fr.json [hunk 1]
```diff
@@ -0,0 +1,4 @@
+{
```
Comment (wyseguyonline) [inline demo/translations/fr.json]: Do we need this?

Comment (JasonLantz) [inline demo/translations/fr.json]: i may need it for proofing the intl...

**Pull request merged**